pkg/metrics/kubemetrics.go

```diff
@@ -108,15 +108,7 @@ func InitKubeMetrics(clusterCache clustercache.ClusterCache, metricsConfig *Metr
 				metricsConfig:    *metricsConfig,
 			})
 		} else if opts.EmitKubeStateMetricsV1Only {
-			prometheus.MustRegister(KubeNodeCollector{
```

Comment:
- Replaced `KubeNodeCollector` with `KubeNodeCollectorV1Only`, a collector emitting identical metrics to the original except for `kube_node_labels` and `kube_node_status_condition`
- Dropped `KubeNamespaceCollector` as it was emitting the `kube_namespace_labels` metric
- Dropped `KubePodLabelsCollector` as it was emitting the `kube_pod_owner` and `kube_pod_labels` metrics